docs/get_started/raspbian_setup.md

```diff
@@ -36,38 +36,37 @@ Clone the MXNet source code repository to your computer, using ```git```.
     git clone https://github.com/dmlc/mxnet.git --recursive
 ```
 
-Then build the full MxNet library, ensuring the ARM flag is enabled (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the compiled library. You can do this by compiling with the following code:
 ```bash
     cd mxnet
     export ARM=1
+    export USE_OPENCV = 0
     make
 ```
 
-If you are not processing images with MxNet on the Pi you can build MxNet without opencv to minimize the size of the loaded library. You can do this by compiling with the following code:
+Otherwise you can build the full MxNet library: 
 ```bash
     cd mxnet
     export ARM=1
-    export USE_OPENCV = 0
     make
 ```
 
+Whenever you are compiling MXNet for the Pi you need to ensure the "ARM" environment variable is set to 1 (this removes the x86 specific msse flag in the NNVM Submodule's Makefile, making it ARM compatible)
+
 Executing either of these commands creates a library called ```libmxnet.so```.
 
-### Install MXNet for Python On Raspberry Pi
+## Install MXNet for Python On Raspberry Pi
 
 To install python bindings run the following commands:
 
 ```bash
-cd setup-utils
-bash ./install-mxnet-ubuntu-python.sh
+    cd python
+    sudo python setup.py install
 ```
 
-Sometimes you have to run the bash command twice as it may fail the first time.
-
 You are now ready to run MxNet on your Raspberry Pi! 
 
-*Note - Keep in mind loading the entire MxNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so steps such as switching off the GUI and killing concurrently running processes to free RAM may be nessecary to load training data or large models into memory.*
-
+*Note - Keep in mind that loading the entire MXNet library takes up a significant fraction (200MB+) of the Raspberry Pi's limited RAM, so you might have to switch off the GUI and kill any concurrently running processes to free the RAM needed to load training data or large models into memory.*
```

Comment:
***Note***
Remember that the entire MXNet library takes up a significant amount (200 MB or more) of the Raspberry Pi's limited RAM. To load training data or large models into memory, you might have to turn off the GUI and terminate running processes to free the RAM.